inlong-manager/manager-plugins/base/src/main/java/org/apache/inlong/manager/plugin/listener/StartupSortListener.java

```diff
@@ -131,30 +140,24 @@ public ListenerResult listen(WorkflowContext context) throws Exception {
                 return ListenerResult.fail(message);
             }
 
-            boolean isRealTimeSync = InlongConstants.DATASYNC_REALTIME_MODE
-                    .equals(groupResourceForm.getGroupInfo().getInlongGroupMode());
-
             FlinkInfo flinkInfo = new FlinkInfo();
 
             String jobName = Constants.SORT_JOB_NAME_GENERATOR.apply(processForm) + InlongConstants.HYPHEN
                     + streamInfo.getInlongStreamId();
             flinkInfo.setJobName(jobName);
             flinkInfo.setEndpoint(kvConf.get(InlongConstants.SORT_URL));
             flinkInfo.setInlongStreamInfoList(Collections.singletonList(streamInfo));
-            if (isRealTimeSync) {
-                flinkInfo.setRuntimeExecutionMode(InlongConstants.RUNTIME_EXECUTION_MODE_STREAMING);
-            } else {
+            if (isOfflineSync) {
                 flinkInfo.setRuntimeExecutionMode(InlongConstants.RUNTIME_EXECUTION_MODE_BATCH);
+            } else {
+                flinkInfo.setRuntimeExecutionMode(InlongConstants.RUNTIME_EXECUTION_MODE_STREAMING);
             }
             FlinkOperation flinkOperation = FlinkOperation.getInstance();
             try {
                 flinkOperation.genPath(flinkInfo, dataflow);
-                // only start job for real-time mode
-                if (isRealTimeSync) {
-                    flinkOperation.start(flinkInfo);
-                    log.info("job submit success for groupId = {}, streamId = {}, jobId = {}", groupId,
-                            streamInfo.getInlongStreamId(), flinkInfo.getJobId());
-                }
+                flinkOperation.start(flinkInfo);
+                log.info("job submit success for groupId = {}, streamId = {}, jobId = {}", groupId,
+                        streamInfo.getInlongStreamId(), flinkInfo.getJobId());
```

Comment:
It is recommended to add real-time/offline identification information in the log content

Reply:
Make sense. Fixed. PTAL again.